fdbclient/admin_client.go

```diff
@@ -201,6 +202,66 @@ func (client *cliAdminClient) runCommand(command cliCommand) (string, error) {
 	return outputString, nil
 }
 
+func shouldRemoveLogFile(info os.FileInfo, now time.Time) (bool, error) {
+	if info.IsDir() {
+		return false, nil
+	}
+
+	if !strings.HasPrefix(info.Name(), "trace") {
+		return false, nil
+	}
+
+	// If the file is newer than 30 minutes skip it.
+	// We assume that the log shipper as pushed the logs away in under 30 minutes.
+	if info.ModTime().Add(30 * time.Minute).Before(now) {
+		return false, nil
+	}
+
+	// Files from the lib will have the format:
+	// trace.$IP.1.&timestamp...json (or xml)
+	// with this regexp we check for the middle part.
+	isLibFile, err := regexp.Compile(`\.1\.\d{10,}`)
+	if err != nil {
+		return false, err
+	}
+
+	// These files are the one from the fdb lib and will be automatically rotated
+	return isLibFile.MatchString(info.Name()), nil
```

Comment:
That should be the other way e.g. the negative one since we want to remove the cli files not the lib files.